doc/api/fs.md

```diff
@@ -2392,6 +2392,32 @@ changes:
 
 Synchronous lchown(2). Returns `undefined`.
 
+## `fs.lutimes(path, atime, mtime, callback)`
+<!-- YAML
+addded: REPLACEME
+-->
+
+* `path` {string|Buffer|URL}
+* `atime` {number|string|Date}
+* `mtime` {number|string|Date}
+* `callback` {Function}
+  * `err` {Error}
+
+Asynchronous lutimes(2). No arguments other than a possible exception
```

Comment:
We should explain what this method does here and link to the man page for lutimes. I think taking the description from the manual page should be fine:

> fs.lutimes() changes the access and modification times of a file in the same way as fs.utimes, with the difference that if filename refers to a symbolic link, then the link is not dereferenced: instead, the timestamps of the symbolic link are changed.

Or something similar - and other references (like the synchronous version) should reference the decsription.